awscli/s3transfer/crt.py

```diff
@@ -114,7 +119,12 @@ def create_s3_crt_client(
 
     :type target_throughput: Optional[int]
     :param target_throughput: Throughput target in Bytes.
-        Default is 0.625 GB/s (which translates to 5 Gb/s).
+        By default, CRT will automatically attempt to choose a target
+        throughput that matches the system's maximum network throughput.
+        Currently, if CRT is unable to determine the maximum network
+        throughput, a fallback target throughput of 1.25 GB/s
+        (which translates to 10Gb/s) is used. To guarantee a specific target
+        throughput, set a value for this parameter.
```

Comment:
> To guarantee a specific target throughput, set a value for this parameter.

semantic nit; should we use a word like "modify" or "set" instead of "guarantee"? I don't think the CRT can ever guarantee it hits the target.

Reply:
I can change that. I can see how that is misleading.